simpleui-server/src/props-file-reader.ts

```diff
@@ -170,49 +194,53 @@ export class PropsFileReader {
         return value;
     }
 
-    static loadUiPropsFile(uiPropsFile: string, appName: string, webPort: string, mtimeMs: string) {
-        const uiProp = path.parse(uiPropsFile).name;
-        const props = { 'srcFile': uiPropsFile };
-        props['uiProp'] = { 'value': uiProp };
 
+
+
+    static loadUiPropsFile(uiPropsFile: string, mtimeMs: string) {
+        let appName = PropsFileReader.cmdVars.appName;
+        const uiProp = path.parse(uiPropsFile).name;
         const versionProps = PropsFileReader.getSoftwareVersionStrings(`/var/www/${appName}/version.txt`);
-        props['uiVersionLong'] = { 'value': versionProps.uiVersionLong };
-        props['uiVersionShort'] = { 'value': versionProps.uiVersionShort };
-
-        props['propsUrl'] =  { 'value': `/${appName}/${uiProp}/query/props`};
-        props['mtimeMs'] = mtimeMs;
-        props['propsLastRefresh'] = ServerUtil.formatDate(new Date(), 'second');
-        props['fullAppUrl'] =  { 'value': `/${appName}/`};
-        props['selectedIndex'] = { 'value': '0' };
-        props['instance'] = { 'name': 'Simple_UI_Instance' };
-        props['nodejsPort'] = { 'value': '2080'};
-        props['refreshRate'] = { 'value': '1200'};
-        props['logLevel'] = { 'value': '0'};
-        props['sessionExpiration'] = { 'value': '6'};
 
+
+        // why are we using { property: { value: abc }} instead of { property: abc } ?
```

Comment:
from my point of view, there's no reason to do so and I'd just clean it up. @jscarsdaleAtLCE thoughts?

Reply:
Don't know for sure, but guess that's just the way it evolved - at one point the values were probably objects with multiple attributes ('name', 'value', etc.), and over time we may have factored out the 'name'. attribute - need to look at any consumer of versionProps and fix any use of attribute named 'value'.

Reply:
I'll definitely check out `versionProps `.  Should those fixes be made in a different branch? I would like to clean it up but I dont think it's related to this branch

Reply:
Don't have a strong opinion, but if it's nearly trivial, save the effort and just include it here.